### PR TITLE
Enforced utf-8 encoding on imported files

### LINKED ISSUE #4693: Import log should probably force a UTF-8 encoding
### Problem
When trying to import a specific folder I get a UnicodeEncodeError. It is most likely related to the pathname/filename.
Running this command in verbose (`-vv`) mode:

```sh
$ beet -vv -c appdata\Roaming\beets\config.yaml import V:\Lupin\
```

Led to this problem:

```
--- Logging error ---
--- Logging error ---
Traceback (most recent call last):
  File "C:\Users\arian\AppData\Local\Programs\Python\Python311\Lib\logging\__init__.py", line 1113, in emit
    stream.write(msg + self.terminator)
  File "C:\Users\arian\AppData\Local\Programs\Python\Python311\Lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
UnicodeEncodeError: 'charmap' codec can't encode characters in position 14-58: character maps to <undefined>
Call stack:
  File "C:\Users\arian\AppData\Local\Programs\Python\Python311\Lib\threading.py", line 995, in _bootstrap
    self._bootstrap_inner()
  File "C:\Users\arian\AppData\Local\Programs\Python\Python311\Lib\threading.py", line 1038, in _bootstrap_inner
    self.run()
  File "C:\Users\arian\AppData\Local\Programs\Python\Python311\Lib\site-packages\beets\util\pipeline.py", line 311, in run
    out = self.coro.send(msg)
  File "C:\Users\arian\AppData\Local\Programs\Python\Python311\Lib\site-packages\beets\util\pipeline.py", line 170, in coro
    task = func(*(args + (task,)))
  File "C:\Users\arian\AppData\Local\Programs\Python\Python311\Lib\site-packages\beets\importer.py", line 1400, in user_query
    task.choose_match(session)
  File "C:\Users\arian\AppData\Local\Programs\Python\Python311\Lib\site-packages\beets\importer.py", line 861, in choose_match
    session.log_choice(self)
  File "C:\Users\arian\AppData\Local\Programs\Python\Python311\Lib\site-packages\beets\importer.py", line 281, in log_choice
    self.tag_log('skip', paths)
  File "C:\Users\arian\AppData\Local\Programs\Python\Python311\Lib\site-packages\beets\importer.py", line 260, in tag_log
    self.logger.info('{0} {1}', status, displayable_path(paths))
  File "C:\Users\arian\AppData\Local\Programs\Python\Python311\Lib\logging\__init__.py", line 1489, in info
    self._log(INFO, msg, args, **kwargs)
  File "C:\Users\arian\AppData\Local\Programs\Python\Python311\Lib\site-packages\beets\logging.py", line 88, in _log
    return super()._log(level, m, (), exc_info, extra)
Message: <beets.logging.StrFormatLogger._LogMessage object at 0x00000236564C5F90>
Arguments: ()
Sending event: import_task_choice
Sending event: import
Sending event: cli_exit
```

Here's a link to the music files that trigger the bug (if relevant):
https://nxtc.arianw.de/s/m2sQx35S37nG8e9

### Setup

* OS: Windows
* Python version: 3.11.1
* beets version: 1.6.0
* Turning off plugins made problem go away (yes/no): no

My configuration (output of `beet config`) is:

```yaml
lyrics:
    bing_lang_from: []
    auto: yes
    bing_client_secret: REDACTED
    bing_lang_to:
    google_API_key: REDACTED
    google_engine_ID: REDACTED
    genius_api_key: REDACTED
    fallback:
    force: no
    local: no
    sources:
    - google
    - musixmatch
    - genius
    - tekstowo

plugins: embedart scrub chroma lyrics lastgenre discogs plexupdate VGMplug
directory: F:\Music
library: C:\Users\arian\AppData\Roaming\beets\library.db
art_filename: albumart
threaded: yes

clutter:
- Thumbs.DB
- .DS_Store
- '*.jpg'
- '*.png'

paths:
    default: $albumartist/$album%aunique{}/$disc.$track - $title
    singleton: Non-Album/$artist - $title
    comp: Various Artists/$album%aunique{}/$disc.$track - $title
    albumtype_soundtrack: Soundtracks/$album/$disc.$track - $title

import:
    write: yes
    copy: no
    move: yes
    resume: ask
    incremental: no
    quiet_fallback: skip
    timid: no
    log: C:\Users\arian\AppData\Roaming\beets\beet.log

replace:
    ^\.: _
    '[\x00-\x1f]': _
    '[<>:"\?\*\|]': _
    '[\xE8-\xEB]': e
    '[\xEC-\xEF]': i
    '[\xE2-\xE6]': a
    '[\xF2-\xF6]': o
    '[\xF8]': o
    \.$: _
    \s+$: ''
web:
    host: 0.0.0.0
    port: 8337
scrub:
    auto: yes
plex:
    host: px.arianw.de
    port: 443
    token: REDACTED
    secure: yes
    library_name: Music
    ignore_cert_errors: no
VGMplug:
    lang-priority: en, ja-latn, ja
    source_weight: 0.0
    artist-priority: composers,performers,arrangers
chroma:
    auto: yes
embedart:
    maxwidth: 0
    auto: yes
    compare_threshold: 0
    ifempty: no
    remove_art_file: no
    quality: 0
discogs:
    apikey: REDACTED
    apisecret: REDACTED
    tokenfile: discogs_token.json
    source_weight: 0.5
    user_token: REDACTED
    separator: ', '
    index_tracks: no
lastgenre:
    whitelist: yes
    min_weight: 10
    count: 1
    fallback:
    canonical: no
    source: album
    force: yes
    auto: yes
    separator: ', '
    prefer_specific: no
    title_case: yes

```

### PR DESCRIPTION
**Pull request by elyang0214**
## Description

Fixes #4693. 
To address the bug outlined in issue #4693, we added a utf-8 encoding argument to the import_files function in the commands.py file.

## To Do

- [x] Documentation. (If you've add a new command-line flag, for example, find the appropriate page under `docs/` to describe it.)
- [x] Changelog. (Add an entry to `docs/changelog.rst` near the top of the document.)
- [x] Tests. (Encouraged but not strictly required.)

### CODE REVIEW

File: docs/reference/cli.rst [hunk 1]
```diff
@@ -172,7 +172,7 @@ Optional command flags:
 
     Just point the ``beet import`` command at a directory of files that are
     already catalogged in your library. Beets will automatically detect this
-    situation and avoid duplicating any items. In this situation, the "copy
+    situation and avoid duplicating any items. A UTF-8 encoding will be enforced on your imported file. In this situation, the "copy
```
Comment (sampsyo) [inline docs/reference/cli.rst:175]: I actually don't think this needs to be in the docs—to be perfectly honest, 99.9% of users will now no longer need to think about the encoding of the file. (Also, this sentence occurs in the middle of a discussion on a different topic.)

File: docs/changelog.rst [hunk 2]
```diff
@@ -11,6 +11,8 @@ for Python 3.6).
 
 New features:
 
+* Added UTF-8 encoding enforcement to imported files in `beets/beets/ui/commands.py`.
```
Comment (sampsyo) [inline docs/changelog.rst:14]: To be more specific: The `--from-logfile` now parses log files using the UTF-8 encoding, regardless of the system's configured locale.

(To be clear, this doesn't have anything to do with music files themselves, for which a text encoding is irrelevant!)

**Review by sampsyo — CHANGES_REQUESTED** [review]
Thanks for getting this started! Here are a couple of small suggestions about the documentation.

File: docs/reference/cli.rst (hunk 1 above)
Comment (elyang0214) [reply docs/reference/cli.rst:175]: Understood, I can incorporate that change!

File: docs/changelog.rst (hunk 2 above)
Comment (elyang0214) [reply docs/changelog.rst:14]: I see, I'll update the documentation to be more accurate.

**Review by sampsyo — APPROVED** [review]
Looks right to me! Many thanks!!

**Pull request merged**